2009-02-06  Ulrich Drepper  <drepper@redhat.com>

	[BZ #9736]
	* locales/el_CY: Fix frac_digits and int_frac_digits.
	* locales/el_GR: Likewise.
	Patch by Clint Adams <schizo@debian.org>.

diff --git a/localedata/locales/el_CY b/localedata/locales/el_CY
index 7213924..3117577 100644
--- a/localedata/locales/el_CY
+++ b/localedata/locales/el_CY
@@ -51,8 +51,8 @@ mon_thousands_sep     "<U002E>"
 mon_grouping          3
 positive_sign         ""
 negative_sign         "<U002D>"
-int_frac_digits       4
-frac_digits           4
+int_frac_digits       2
+frac_digits           2
 p_cs_precedes         0
 p_sep_by_space        0
 n_cs_precedes         1
diff --git a/localedata/locales/el_GR b/localedata/locales/el_GR
index ce608a8..bb1db14 100644
--- a/localedata/locales/el_GR
+++ b/localedata/locales/el_GR
@@ -70,8 +70,8 @@ mon_thousands_sep     "<U002E>"
 mon_grouping          3
 positive_sign         ""
 negative_sign         "<U002D>"
-int_frac_digits       4
-frac_digits           4
+int_frac_digits       2
+frac_digits           2
 p_cs_precedes         0
 p_sep_by_space        0
 n_cs_precedes         1
